UML editor/src/main/java/codecain/UMLClassInfo.java

```diff
@@ -0,0 +1,89 @@
+package codecain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The UMLClassInfo class represents the details of a UML class, including its name, fields, and methods.
+ * It provides methods to retrieve and manage the fields and methods of the class.
+ */
+public class UMLClassInfo {
```

Comment:
Nice addition to our code. We had a hard time with the fields, and methods in sprint one, especially displaying them.